packages/terra-application-utility/src/ApplicationHeaderUtility.jsx

```diff
@@ -49,6 +54,7 @@ const ApplicationHeaderUtility = ({
     initialSelectedKey={initialSelectedKey}
     title={title}
     accessory={accessory}
+    role={menuRole}
```

Comment:
```suggestion
    menuRole={menuRole}
```

Reply:
[Changed to menurole](https://github.com/cerner/terra-framework/pull/501/commits/131aaf225ec1730dafcfecf63f4d79d0857c6bfa)

Reply:
update to camelCase - `menuRole`